### PR TITLE
python3Packages.pubnubsub-handler: init at 1.0.9

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
PubNub subscription between PubNub and Home Assistant.

https://github.com/w1ll1am23/pubnubsub-handler

This is a Home Assistant dependency.

This PR includes the needed dependencies for `pubnubsub-handler`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pubnub/default.nix [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  checkInputs = [
+    aiohttp
+    pycryptodomex
+    pytest-asyncio
+    pytestCheckHook
+    pytest-vcr
+
+  ];
+
+  # Some tests don't pass with recent releases of tornado/twisted
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pubnub/default.nix:42]: Is this a test issue or a problem with the code itself? Maybe we should require an older version to not ship broken packages.

Comment (jonringer) [reply pkgs/development/python-modules/pubnub/default.nix:42]: from the comment, it sounds like the tornado updates broke the test suite.

This is a wide spread issue within the python ecosystem in general.

Generally upstreams will just pin tornado or other dependencies to small range, at the cost of being potentially incompatible with other package installations

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109556` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
9 packages built:
</summary>
<ul>
<li>python37Packages.pubnub</li>
<li>python37Packages.pubnubsub-handler</li>
<li>python37Packages.pytest-vcr</li>
<li>python38Packages.pubnub</li>
<li>python38Packages.pubnubsub-handler</li>
<li>python38Packages.pytest-vcr</li>
<li>python39Packages.pubnub</li>
<li>python39Packages.pubnubsub-handler</li>
<li>python39Packages.pytest-vcr</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109556` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
9 packages built:
</summary>
<ul>
<li>python37Packages.pubnub</li>
<li>python37Packages.pubnubsub-handler</li>
<li>python37Packages.pytest-vcr</li>
<li>python38Packages.pubnub</li>
<li>python38Packages.pubnubsub-handler</li>
<li>python38Packages.pytest-vcr</li>
<li>python39Packages.pubnub</li>
<li>python39Packages.pubnubsub-handler</li>
<li>python39Packages.pytest-vcr</li>
</ul>
</details>

**Pull request merged**